bliss/catalog.py

```diff
@@ -13,11 +13,11 @@
 
 
 def convert_mag_to_nmgy(mag):
-    return 10 ** ((22.5 - mag) / 2.5)
+    return 10 ** ((22.5 - mag) / 2.5) * 3631
 
 
 def convert_nmgy_to_mag(nmgy):
-    return 22.5 - 2.5 * torch.log10(nmgy)
+    return 22.5 - 2.5 * torch.log10(nmgy / 3631)  # dc2
```

Comment:
This isn't a valid conversion from nanomaggies to magnitude if you divide: better to keep this within a case study  and to give the function a new name